src/routes/Edit/VideoSegmentItem.tsx

```diff
@@ -33,6 +33,7 @@ const VideoSegmentItem = ({
               style={{ margin: 0 }}
               circular
               icon={canEdit ? 'edit' : 'lock'}
+              color={active ? 'green' : 'grey'}
```

Comment:
Was it intentional to make the non-active buttons dark grey instead of the current light grey?

Ex: 

Current button (no hover):
<img width="311" alt="Screen Shot 2021-01-18 at 11 00 42 AM" src="https://user-images.githubusercontent.com/12631935/104954117-89581500-597c-11eb-9261-d85de52c6aed.png">

Current button (hover):
<img width="311" alt="Screen Shot 2021-01-18 at 11 04 05 AM" src="https://user-images.githubusercontent.com/12631935/104954418-10a58880-597d-11eb-9c3c-9f71174a082c.png">


New button (no hover):
<img width="312" alt="Screen Shot 2021-01-18 at 11 00 33 AM" src="https://user-images.githubusercontent.com/12631935/104954140-8f4df600-597c-11eb-80cd-e257d387c364.png">

New button (hover):
<img width="316" alt="Screen Shot 2021-01-18 at 11 04 14 AM" src="https://user-images.githubusercontent.com/12631935/104954402-097e7a80-597d-11eb-8ee5-5732ac294cc9.png">


The main disadvantage I see is that the "hover" style is harder to see with the dark grey button currently. 

Just wanted to point this out, however I leave the final style choice up to you.

Reply:
I guess that was unintentional. I will fix it in the next round.